app/controllers/bookings_controller.rb

```diff
@@ -27,11 +36,15 @@ def destroy
     # Changed path back to "bookings path""
   end
 
+  # def update
+  #   @booking.update(booking_params)
+  #   redirect_to booking_path(@booking)
+  # end
+
   private
 
   def booking_params
     params.require(:booking).permit(:starting_date, :ending_date, :status)
-    # permit status?
```

Comment:
Kept it in because it is needed for updating the status of booking to accepted and rejected. You can kind of see the logic in the show/html/erb of booking